### DISCUSSION TITLE
Enhancing Documentation with VitePress

### DISCUSSION THREAD

**Discussion by harilvfs** [ChrisTitusTech/linutil#968] [Ideas]
First of all, I’d like to express my gratitude to Christitus and all the contributors for creating such an amazing tool that helps Linux users in so many ways.

I’ve taken a look at the Linutil docs, and I see it’s built with Hugo, which I personally love. However, I wanted to share some thoughts—please don’t take this the wrong way, as it’s just my perspective. While the docs provide valuable information, I believe there’s room for improvement in terms of user experience and presentation. A more refined and user-friendly documentation could really help beginners get up to speed with Linutil.

One suggestion I have is to consider using VitePress for the docs. If you haven’t heard of it, you can check it out here: [VitePress](https://vitepress.dev/). I believe VitePress could make the documentation more visually appealing, easier to read, and more accessible for users.

This is just a small suggestion from my side, and I’d be happy to help implement it if any of the maintainers agree with this approach. It will take time, but I’m willing to contribute to the development if needed.

Thanks again for this fantastic tool!

By the way, sorry if this sounds off—this description was refined using an LLM, but the words are all my own.

**Comment by koibtw** [comment]
This seems cool but I'd rather use the [Rust mdBook](https://rust-lang.github.io/mdBook). We just need someone to write the documentation now...

**Comment by harilvfs** [comment]
Alright, I haven’t heard of that before, but it seems quite clean. I’ll take a look at it, and if I can build it, I’ll let you know.
Thank you!

**Comment by harilvfs** [comment]
@adamperkowski As you mentioned, I have built the documentation website using mdBook. Please take a look and share any feedback or suggestions for improvements. I would be happy to work on it further! [Check out the demo here](https://harilvfs.github.io/rustmdbook/). I initially set it up in my repository for testing. If you approve of the documentation, I’ll proceed with creating a PR for the Linutil docs.

**Reply by koibtw** [reply]
You can open a PR now, I'll review it and add suggestions.
